core/src/main/java/org/infinispan/configuration/cache/TracingConfiguration.java

```diff
@@ -26,4 +30,33 @@ protected TracingConfiguration(AttributeSet attributes) {
    public boolean enabled() {
       return attributes.attribute(ENABLED).get();
    }
+
+   public boolean container() {
+      return attributes.attribute(CONTAINER).get();
```

Comment:
In my opinion, these method should check for `enabled()`; otherwise, `container()` may return a different value than `enabled(CONTAINER)`

Reply:
I agree